btcstaking-tracker/stakingeventwatcher/stakingeventwatcher.go

```diff
@@ -426,7 +427,7 @@ func (sew *StakingEventWatcher) watchForSpend(spendEvent *notifier.SpendEvent, t
 		return
 	}
 
-	schnorrSignature, err := tryParseStakerSignatureFromSpentTx(spendingTx, td)
+	_, err := tryParseStakerSignatureFromSpentTx(spendingTx, td)
```

Comment:
this function actually checks whether stake spending transaction is unbonding transaction registered to Babylon. 

So imo we can leave it but we should send inclusion proof through `sew.reportUnbondingToBabylon(quitCtx, delegationId, spendingTx, proof)` even if it returns error (as now we have possibility to report even invalid unbondings to Babylon)

So the only difference between error case and non err case is metrics which we are incrementing.